### ISSUE TITLE
(BUG)Upvote and downvote not shown on Alexandrite

### ISSUE THREAD

**Issue by dnet890** [sheodox/alexandrite#104]
Hi
I use feddit.nl and I checked that upvote and downvote are gone. But, I checked on main instance it shows upvotes and downvotes properly. Is it a bug?

<img width="636" alt="2024-07-16_18-22_1" src="https://github.com/user-attachments/assets/2012927b-46da-40c8-aa79-da8e00bb4b99">
<img width="813" alt="2024-07-16_18-22" src="https://github.com/user-attachments/assets/1460a230-b1ba-4c30-8ee7-464ea4d67e90">
<img width="908" alt="2024-07-16_18-21" src="https://github.com/user-attachments/assets/ce21dfdb-e5b5-45c9-b45b-1f6ad557abb3">

**Comment by ramast** [comment]
you mean upvote/downvote count right? Am also having this problem

**Comment by sheodox** [comment]
Are you still seeing this issue? I just checked and they're showing for me.

**Comment by ramast** [comment]
Yes, issue still happening for me. Could it be depending on the instance? my instance is sh.itjust.works

**Comment by sheodox** [comment]
Have you tried disabling any extensions? What browser? It's working fine for me.

![Image](https://github.com/user-attachments/assets/614c3efe-a5fd-438c-b0fa-ff5737d60e64)

**Comment by ramast** [comment]
My webbrowser is firefox.
this is strange: If I open private window or switch to guest account
upvotes show normally. As soon as I switch back to my sh.itjust.works
account, upvotes disappear again.

Do you have sh.itjust.works account to test with?

**Comment by sheodox** [comment]
Yes I tried with an account. Do you have an ad blocker? I wonder if there's a custom ad blocking rule you accidentally set up.

**Comment by ramast** [comment]
I don't have ad blockers but anyway to be sure its not the browser, I
tested it in chromium (which I rarely use and have no addons installed)
still same problem.
I opened browser console but didn't find any related error.
I have web development background so if there is something I could do to
help debugging this please let me know

**Comment by sheodox** [comment]
Oooooh,  go to the settings page and under the "Lemmy" tab, is  "Show scores" checked?

**Comment by ramast** [comment]
It was not checked.
Show upvotes and downvotes were checked but not show scores.

Checking it solved the problem for me but I feel we still need a way to
inform users that this option is needed for Alexandrite to work because in
default UI I can the score just fine

**Comment by sheodox** [comment]
I just checked on lemmy-ui and when I disabled that setting all the scores went away. I'm not sure why you're still seeing them with the option off to be honest. That seemed like the whole point of that option in particular.

![Image](https://github.com/user-attachments/assets/03622ec4-eb3e-45aa-9962-59be76693988)

**Comment by ramast** [comment]
Maybe its something sh.itjust.works specific.

Thank you so much for looking into this.
Hopefully if someone else encounter this problem, they'll find this github
issue and follow your solution.

Cheers

**Comment by sheodox** [comment]
If someone else is having issues I can reopen.
